Translation corrections (#507)
07th-mod/higurashi-console-arcs: Update/kage_011.txt

	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200331", 256, TRUE);
	OutputLine(NULL, "「くそ真面目に生きたって、バカ正直に頑張ったって、もっと世の中にずるく賢く立ち回って、偉くなってる人がいるんだから。",
			NULL, "\"Even if you live so earnestly, working so hard over a foolish notion of integrity, there are others out there who act shrewder and smarter, and thus rise up in the world.", Line_WaitForInput);
	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200332", 256, TRUE);
	OutputLine(NULL, "その分自分が損をするだけじゃないかって…そう、思ってました」",
			NULL, " She was just hurting herself doing all of that... at least, that's what I used to think.\"", Line_Normal);
	ClearMessage();

//まどかrvS08/29/321200333.「今でも、少しだけそう思います」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a4856e>まどか</color>", NULL, "<color=#a4856e>Madoka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200333", 256, TRUE);
	OutputLine(NULL, "「今でも、少しだけそう思います」",
			NULL, "\"No, even now, I still think that just a little.\"", Line_Normal);
	ClearMessage();

//rでも、とまどかさんは顔を上げる。…どこか、晴れやかな表情だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも、とまどかさんは顔を上げる。…どこか、晴れやかな表情だった。",

> 			NULL, "\"But\", Madoka-san said, raised her head. ...And she gallantly looked off in the distance.", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"But\", Madoka-san said while raising her head... and gallantly looking off in the distance.", Line_Normal);
```

//rあの心のうちに熱情を秘めるような人が、歳を取ってからの子どもを喜ばないはずがない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あの心のうちに熱情を秘めるような人が、歳を取ってからの子どもを喜ばないはずがない。",

> 			NULL, "A person with passion in their heart like him would never not be happy to have a child just because they've grown old.", Line_Normal);

Reviewer: ```suggestion
			NULL, "A person with passion in their heart like him would never be unhappy to have a child just because they've grown old.", Line_Normal);
```

			NULL, "\"That's ridiculous...", Line_WaitForInput);
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000983", 256, TRUE);
	OutputLine(NULL, "二重人格なんてそれこそ、空想の中の話でしょう？",

> 			NULL, " Aren't split personalities all fictional?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Aren't split personalities the sort of thing you read about in stories?", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s08/11/321100931", 256, TRUE);
	OutputLine(NULL, "「いわゆる精神疾患の一種ですがね、激しい感情を抱き続ける自分自身を嫌悪するあまり、脳がそれを自分以外の誰かだと判断して、分離しちゃうってワケです。",

> 			NULL, "\"It's a type of mental illness, where the person in question has a strong enough sense of self-loathing for their negative emotions that their brain can rationalize that part of itself as a separate entity to segregate itself from it. ", GetGlobalFlag(GLinemodeSp));

Reviewer: ```suggestion
			NULL, "\"It's a type of mental illness, where the person in question has a strong enough sense of self-loathing toward their negative emotions that the brain can rationalize that part of itself as a separate entity to segregate itself from it. ", GetGlobalFlag(GLinemodeSp));
```

//r人間なら誰しも、英雄願望…自分が物語の主人公やヒロインのように、理想的な人物像でありたいと願う気持ちがある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "人間なら誰しも、英雄願望…自分が物語の主人公やヒロインのように、理想的な人物像でありたいと願う気持ちがある。",
			NULL, "Much like how every human being has an innate desire to be great... to be the hero or heroine of their own story, they also wish to be their ideal self.", Line_Normal);
	ClearMessage();

//rとはいえ、たいていの場合は現実とのズレに失望し、妥協して社会的に自制した思考と言動を身につけてゆく。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "とはいえ、たいていの場合は現実とのズレに失望し、妥協して社会的に自制した思考と言動を身につけてゆく。",
			NULL, "However, in most cases, people get disappointed by the gap between reality and their ideals, compromise, and learn what ideas and actions are socially acceptable or not.", Line_Normal);
	ClearMessage();

//r…それでも、あまりにもその差が激しい場合、現実を受容することを拒絶し、自らが作り出した理想に埋没して社会との隔絶をはかる。それが、『現実逃避』だ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…それでも、あまりにもその差が激しい場合、現実を受容することを拒絶し、自らが作り出した理想に埋没して社会との隔絶をはかる。それが、『現実逃避』だ。",

> 			NULL, "...However, if that gap is too severe, the person can refuse to accept reality, and isolate theirself from society by immersing in their ideals. That's \"escapism.\" ", Line_Normal);

Reviewer: ```suggestion
			NULL, "...However, if that gap is too severe the person can refuse to accept reality, isolating themself from society by immersing in their ideals. That's \"escapism.\" ", Line_Normal);
```

	ClearMessage();

//rそして、それは最小の社会である、『個人』の中でも起こりえることがあるらしい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、それは最小の社会である、『個人』の中でも起こりえることがあるらしい。",

> 			NULL, "And that's possibile even within an \"individual\", the smallest society there is.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And that's possible even within an \"individual\", the smallest society there is.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s08/11/321100937", 256, TRUE);
	OutputLine(NULL, "「で、ここで出てくるのが、例の薬です。",
			NULL, "\"So, this is where that medicine comes in.", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s08/11/321100938", 256, TRUE);
	OutputLine(NULL, "以前は公由夏美の激しい感情を、その薬で押さえつけているって話でした」",

> 			NULL, " The story we've been told is that Natsumi Kimiyoshi suppressed her intense emotions with that medicine.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " The story we've been told is that Natsumi Kimiyoshi has been taking that medicine to suppress her intense emotions.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s08/11/321100941", 256, TRUE);
	OutputLine(NULL, "「で、その薬ってやつが黒いほうの衝動を解消する効能ではなく、むしろ白いほうで溜まりそうになっていた衝動を黒いほうに押しやって……」",

> 			NULL, "\"So, maybe this medicine didn't actually eliminate the impulses of her dark side, but instead pushed those impulses that were close to accumulating on her light side onto her dark side...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"So, maybe this medicine didn't actually eliminate the impulses of her dark side, but instead pushed those piling impulses away from her light side and onto her dark side...\"", Line_Normal);
```

	ClearMessage();

//大石rvS08/11/321100942.「見えないようにふたをする役割を果たしていたのだとしたら、…どうなりますか？」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s08/11/321100942", 256, TRUE);
	OutputLine(NULL, "「見えないようにふたをする役割を果たしていたのだとしたら、…どうなりますか？」",

> 			NULL, "\"If it served as a sort of lid so those emotions would never be seen... what would happen?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"If it served as a sort of lid to keep those emotions from ever being seen... what would happen?\"", Line_Normal);
```

	ClearMessage();

//赤坂rvS08/13/321000986.「…黒に溜まった感情は、どんどん増大していきます！kvS08/13/321000987.　そして、何かのきっかけで人格が反転したとき、それらが一気に爆発する…っ！！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000986", 256, TRUE);
	OutputLine(NULL, "「…黒に溜まった感情は、どんどん増大していきます！",

> 			NULL, "\"...The emotions accumulated on the dark side would grow, bigger and bigger!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...The emotions accumulating on the dark side would grow, bigger and bigger!", Line_WaitForInput);
```

//r決まってる…決まってる、あいつだよッッ！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "決まってる…決まってる、あいつだよッッ！！",
			NULL, "That's obvious... It's her, of course!!", Line_Normal);
	ClearMessage();

//rまだあいつ、たいした怪我もなく残ってるじゃない！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まだあいつ、たいした怪我もなく残ってるじゃない！",

> 			NULL, "She wasn't badly injured, so she's still out there!", Line_Normal);

Reviewer: ```suggestion
			NULL, "She wasn't that badly injured, so she's still out there!", Line_Normal);
```

	ClearMessage();

//rワタシの幸せ、横取っていくあいつが——！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ワタシの幸せ、横取っていくあいつが——！！",

> 			NULL, "She, who's taking my happiness from me...!!", Line_Normal);

Reviewer: ```suggestion
			NULL, "The one taking my happiness from me...!!", Line_Normal);
```
